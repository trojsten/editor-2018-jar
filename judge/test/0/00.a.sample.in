VECTOR_INT:
intv
4
1
2
3
4
INT:
intt
10
VECTOR_STR:
strv
2
abcd
sf
STR:
strr
ddgf
VECTOR_FLOAT:
floatv
3
0.100000
0.200000
0.300000
FLOAT:
floatt
0.001000
