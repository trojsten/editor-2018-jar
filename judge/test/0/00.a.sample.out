{"intv": [1, 2, 3, 4], "strv": ["abcd", "sf"], "floatv": [0.1, 0.2, 0.3], "intt": 10, "floatt": 0.001, "strr": "ddgf"}